support-workers/src/main/scala/com/gu/support/workers/lambdas/SendThankYouEmail.scala

```diff
@@ -36,12 +37,21 @@ class SendThankYouEmail(
   ): FutureHandlerResult = {
     val emailBuilder = new EmailBuilder(
       services.zuoraService.getMandateIdFromAccountNumber,
+      _ match {
+        case None => Future.successful(None)
+        case Some(n) =>
+          for {
+            response <- services.paperRoundService.agents()
+          } yield response.data.agents.find(_.refId == n)
+      },
       services.promotionService,
       Configuration.stage,
     )
 
     for {
-      emailFields <- emailBuilder.buildEmail(state)
+      emailFields <- emailBuilder.buildEmail(
+        state,
+      )
```

Comment:
can we just leave formatting this how it was?

Reply:
Yes, of course – I had originally added a field here, and when I removed it I didn’t notice I’d left the formatting change.